Dockerfile

```diff
@@ -4,27 +4,34 @@ FROM $DEPENDENCY_IMAGE AS builder
 
 COPY . ./
 
-RUN  \
-    python ./build_with_conan.py --release --parallel 4\
+RUN python3 ./build_with_conan.py --release --parallel 4 \
     && cp build/silo_test . \
     && cp build/siloApi .
 
 
-FROM alpine:3.18 AS server
+FROM ubuntu:24.04 AS server
```

Comment:
@corneliusroemer, I think that it is better to use small images with as few dependencies as possible to reduce the surface for potential security vulnerabilities.